Include/object.h

```diff
@@ -141,7 +141,7 @@ static inline PyTypeObject* _Py_TYPE(const PyObject *ob) {
 
 
 static inline int _Py_IS_TYPE(const PyObject *ob, const PyTypeObject *type) {
-    return ob->ob_type == type;
```

Comment:
Hm, There have any benefit from this change?

Reply:
With this only, only Py_TYPE() has to be updated, Py_IS_TYPE() remains unchanged.